src/zenml/new/pipelines/pipeline.py

```diff
@@ -285,6 +240,35 @@ def from_model(cls, model: "PipelineResponseModel") -> "Pipeline":
 
         return load_pipeline(model=model)
 
+    @property
+    def model(self) -> "PipelineResponseModel":
```

Comment:
Yep was thinking the same, but this is the best thing I could come up with.
- `to_model()` would be more consistent with other classes we have (schemas, etc.) but it's kinda wrong since you don't convert the pipeline to a model, but fetch the model from the DB instead
- The initial name `_get_registered_model()` is accurate but very verbose and therefore not ideal for user-facing functionality

Reply:
Overall, I actually think we should expose the models to users more since they are basically the Python API equivalent of swagger responses. If you want to query ZenML data programmatically via the Python API, the `Client` and the response models are the most important concepts you need to know about.

Reply:
alternative suggestion would be  version? This is technically a version of a pipeline?